serving/serving.go

```diff
@@ -445,58 +445,73 @@ func (serv *FeatureServer) getVectorTable(ctx context.Context, fv *metadata.Feat
 	return vectorTable, nil
 }
 
-func (serv *FeatureServer) ResourceLocation(ctx context.Context, req *pb.TrainingDataRequest) (*pb.ResourceFileLocation, error) {
-	// TODO: Modify this method to return the location of any resource within Featureform
-	// - This will require a change in the input to be a ResourceID (name, variant, and type)
-	// - Modifications to the Offline Store interface to pull the latest file location of a resource
+func (serv *FeatureServer) GetResourceLocation(ctx context.Context, req *pb.ResourceIdRequest) (*pb.ResourceLocation, error) {
+	// Pulls the location of a resource from the provider
 
-	id := req.GetId()
-	name, variant := id.GetName(), id.GetVersion()
-	serv.Logger.Infow("Getting the Resource Location:", "Name", name, "Variant", variant)
+	name := req.GetName()
+	variant := req.GetVariant()
+	resourceType := req.GetType()
+	serv.Logger.Infow("Getting the Resource Location:", "Name", name, "Variant", variant, "Type", resourceType)
 
-	tv, err := serv.Metadata.GetTrainingSetVariant(ctx, metadata.NameVariant{Name: name, Variant: variant})
-	if err != nil {
-		return nil, errors.Wrap(err, "could not get training set variant")
+	var location string
+	var err error
+	if resourceType == 2 {
```

Comment:
Can we use a constant to avoid having this magic number in the conditional logic?